Dependencies/UrlLib/Source/Android/UrlRequest.cpp

```diff
@@ -65,8 +65,8 @@ namespace UrlLib
                 try
                 {
                     Uri uri{Uri::Parse(m_url.data())};
-                    std::string scheme{uri.getScheme()};
-                    if (scheme == "app")
+                    String scheme{uri.getScheme()};
+                    if (scheme != nullptr && (std::string)scheme == "app")
```

Comment:
If we did what I mentioned above, this could could look like:
```c++
std::optional<std::string> scheme{uri.getScheme()};
if (scheme == "app")
```
Maybe this is better? @bghgary thoughts?